### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by mootz12**
cc: @carlosdimatteo

### CODE REVIEW

**Review by drew-boy — APPROVED** [review]
gudjob

File: src/contexts/wallet.tsx [hunk 1]
```diff
@@ -129,9 +131,20 @@ export const WalletProvider = ({ children = null as any }) => {
...
       setTxStatus(TxStatus.SIGNING);
-      let result = await signTransaction(xdr, { networkPassphrase: network.passphrase });
-      setTxStatus(TxStatus.SUBMITTING);
-      return result;
+      try {
+        let result = await signTransaction(xdr, { networkPassphrase: network.passphrase });
+        setTxStatus(TxStatus.SUBMITTING);
+        return result;
+      } catch (e: any) {
+        if (e == 'User declined access') {
```
Comment (carlosdimatteo) [inline src/contexts/wallet.tsx:139]: nitpick : ideally use `===`

**Review by carlosdimatteo — APPROVED** [review]
small nitpick otherwise good , i do think we can capitalize in css instead of js but could happen in another pr 👍

File: src/contexts/wallet.tsx (hunk 1 above)
Comment (mootz12) [reply src/contexts/wallet.tsx:139]: will fix in followup PR so we can merge this

**Pull request merged**